prettyconf/loaders.py

```diff
@@ -130,48 +203,89 @@ def __contains__(self, item):
         if self.configs is None:
             self._parse()
 
-        return item in self.configs
+        return self.var_format(item) in self.configs
 
     def __getitem__(self, item):
         if self.configs is None:
             self._parse()
 
-        return self.configs[item]
+        return self.configs[self.var_format(item)]
 
 
-class IniFileConfigurationLoader(AbstractFileConfigurationLoader):
-    default_section = "settings"
-    patterns = ("*.ini", "*.cfg")
+class RecursiveSearch(AbstractConfigurationLoader):
```

Comment:
This loader looks like a "composite" loader (https://en.wikipedia.org/wiki/Composite_pattern).

Why we don't implement it like (pseudo code):

```python
loader = RecursiveSearch(
    starting_path=...,
    loaders=[
        EnvFile,
        IniFile,
    ]
)
```

It's just a suggestion...

Reply:
I like this very much, but `IniFile` doesn't work with globs, in a future release we could deprecate the glob (*.ini, *.cfg) behavior and implement it this way.